ui/helpers/constants/settings.js

```diff
@@ -512,6 +512,13 @@ const SETTINGS_CONSTANTS = [
     route: `${EXPERIMENTAL_ROUTE}#redesigned-confirmations`,
     icon: 'fas fa-flask',
   },
+  {
+    tabMessage: (t) => t('experimental'),
+    sectionMessage: (t) => t('redesignedTransactionsEnabledToggle'),
+    descriptionMessage: (t) => t('redesignedTransactionsToggleDescription'),
+    route: `${EXPERIMENTAL_ROUTE}#redesigned-transactions`,
+    icon: 'fas fa-flask',
+  },
```

Comment:
the way our settings works is not intuitive. The order at which they appear here matters. 

We should actually move this setting and the one above, above the experimental snap setting since the experimental snap setting is hidden behind a build condition. 

The settingsRefs # anchor we use in the jsx matches the ordering here. 

I started labeling these with comments e.g. 
```
// experimental settingsRefs[1]
```

to help match these accordingly

Reply:
after a second look, I don't think this is a blocker though it is still a good idea to group the settings together

I see there was code that was added 2 weeks ago ([PR](https://github.com/MetaMask/metamask-extension/pull/25672/files#diff-a342e4508644fdc378f3acf43755a2fa484916f3eac050df4e2059927c128709)) to  
 route all experimental toggle settings to this.settingsRefs[0], which maps to the Petnames setting which is at the top of the page. This seems okay to me but is prone to error e.g. if more settings were placed above Petnames and the anchor scrolled down to Petnames when they are not Petnames